native/dlahqr.go

```diff
@@ -37,37 +36,34 @@ import (
 // wr and wi must have length n.
 //
 // z and ldz represent an n×n matrix Z. If wantz is true, the transformations
-// will be applied to the submatrix Z[iloz:ihiz+1,ilo:ihi+1]. If wantz is false,
-// z is not referenced. It must hold that
-//  0 <= iloz <= ilo; ihi <= ihiz < n.
-
-// converged indicates whether Dlahqr computed all the eigenvalues ilo to ihi in
-// a total of 30 iterations per eigenvalue. If converged is false, some
-// eigenvalues have not converged, and wr[index+1:ihi+1] and wi[index+1:ihi+1]
-// contain those eigenvalues which have been successfully computed.
-//
-// If converged is true and wantt is true, on return H[ilo:ihi+1,ilo:ihi+1] will
-// be overwritten by upper quasi-triangular full Schur form with any 2×2
-// diagonal blocks in standard form.
+// will be applied to the submatrix Z[iloz:ihiz+1,ilo:ihi+1] and it must hold that
+//  0 <= iloz <= ilo, and ihi <= ihiz < n.
+// If wantz is false, z is not referenced.
 //
-// If converged is true and wantt is false, the contents of h on return is
-// unspecified.
+// unconverged indicates whether Dlahqr computed all the eigenvalues ilo to ihi
+// in a total of 30 iterations per eigenvalue.
 //
-// If converged is false and wantt is false, on return, the remaining
-// unconverged eigenvalues are the eigenvalues of the upper Hessenberg matrix
-// H[ilo:index+1,ilo:index+1].
+// If unconverged is zero, all the eigenvalues ilo to ihi have been computed and
+// are stored on return in wr[ilo:ihi+1] and wi[ilo:ihi+1]. If wantt is true,
+// on return H[ilo:ihi+1,ilo:ihi+1] will be overwritten by upper
+// quasi-triangular full Schur form with any 2×2 diagonal blocks in standard
+// form. If wantt is false, the contents of h on return is unspecified.
 //
-// If converged is false and wantt is true, then on return
+// If unconverged is positive, some eigenvalues have not converged, and
+// wr[unconverged:ihi+1] and wi[unconverged:ihi+1] contain those eigenvalues
+// which have been successfully computed. If wantt is false, on return the
+// remaining unconverged eigenvalues are the eigenvalues of the upper Hessenberg
+// matrix H[ilo:unconverged,ilo:unconverged]. If wantt is true, then on return
 //  (initial H)*U = U*(final H),   (*)
 // where U is an orthogonal matrix. The final H is upper Hessenberg and
-// H[index+1:ihi+1,index+1:ihi+1] is upper triangular.
+// H[unconverged:ihi+1,unconverged:ihi+1] is upper quasi-triangular.
```

Comment:
I think that reference LAPACK means _quasi_-triangular here, triangular does not make much sense.